depthai_sdk/src/depthai_sdk/oak_camera.py

```diff
@@ -123,6 +124,7 @@ def create_camera(self,
                                encode=encode,
                                rotation=self._rotation,
                                replay=self.replay,
+                               name=name,
```

Comment:
Add the name to the comments underneath the method (both inside oak_camera and CameraComponent), so it gets auto-populated to the documentation. Also perhaps we could add this functionality to other components?